include/eve/concept/simd.hpp

```diff
@@ -0,0 +1,40 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/arch/spec.hpp>
+#include <eve/concept/scalar.hpp>
+#include <eve/traits/element_type.hpp>
+
+namespace eve
+{
+  inline namespace EVE_ABI_NAMESPACE
+  {
+    template<scalar Type, typename Size> struct wide;
+  }
+
+  namespace detail
+  {
+    template<typename T> struct is_wide { static constexpr bool value = false; };
+
+    template<scalar T, typename N>
+    struct is_wide<wide<T,N>> { static constexpr bool value = true; };
+  }
+
+  template<typename T>
+  concept simd = detail::is_wide<T>::value;
```

Comment:
Should we just move instance of to main detail? We have it in algo, we needed it for aligned pointer, now here?

Reply:
Wait. Is logical<simd> - simd?

Reply:
Per #1144, logical<wide< is a simd_value but not simd
simd is basically the builtin stuff+product type as we decided in the #1144 discussion

Reply:
I'll move instance_of into meta.hpp